### PR TITLE
use more icons in player board

### PR DESCRIPTION
**Pull request by zeitlinger**
- improve display of taken players in faction wheel
- use icons to show research progress in player area
- show taken sectors and power values of structures in and outside federations for all players
- satellite image is from https://upload.wikimedia.org/wikipedia/commons/2/27/FP_Satellite_icon.svg

image

image

### CODE REVIEW

File: src/components/PlayerBoard/Info.vue [hunk 1]
```diff
@@ -99,9 +114,17 @@ export default class BuildingGroup extends Vue {
...
 
-  get researchOverview (): string {
-    const r = this.data.research;
-    return `Sat: ${this.data.satellites} TF: ${r.terra} Nav: ${r.nav} AI: ${r.int} GP: ${r.gaia} Eco: ${r.eco} Sci: ${r.sci}`;
+  researchType (index: number): string {
+    return ResearchField.values()[index];
+  }
+
+  research (index: number): number {
+    return this.data.research[ResearchField.values()[index]];
```
Comment (coyotte508) [inline src/components/PlayerBoard/Info.vue:122]:
```suggestion
    return this.data.research[this.researchType(index)];
```

File: src/components/PlayerBoard/Info.vue [hunk 2]
```diff
@@ -99,9 +114,17 @@ export default class BuildingGroup extends Vue {
     return parseInt(this.player.income.substr(index));
   }
 
-  get researchOverview (): string {
-    const r = this.data.research;
-    return `Sat: ${this.data.satellites} TF: ${r.terra} Nav: ${r.nav} AI: ${r.int} GP: ${r.gaia} Eco: ${r.eco} Sci: ${r.sci}`;
+  researchType (index: number): string {
```
Comment (coyotte508) [inline src/components/PlayerBoard/Info.vue:117]:
```suggestion
  researchType (index: number): ResearchField {
```

**Comment by coyotte508** [comment]
It would probably make sense to have white text in some of the research fields

**Comment by zeitlinger** [comment]
thanks for fixing the text color in the research board right away :smile:

**Pull request merged**